pkg/meta/dump.go

```diff
@@ -382,7 +382,9 @@ func loadEntries(r io.Reader, load func(*DumpedEntry), addChunk func(*chunkKey))
 
 	progress := utils.NewProgress(false)
 	bar := progress.AddCountBar("Loaded entries", 1) // with root
-	dm = &DumpedMeta{}
+	dm = &DumpedMeta{
+		Counters: &DumpedCounters{},
```

Comment:
The file loaded is not a valid metadata backup if it's `dm.Counters` is nil.

Reply:
We'd better check it after `progress.Done` and before the final log.

Reply:
Okay, this is more rigorous